bin/env-var-docs-check-vars

```diff
@@ -2,20 +2,14 @@
 
 # DOC: Runs env-var-docs/{check_vars_documented,run_regex_tests}.py.
 
-source bin/lib.sh
-
-if [[ ! -d env-var-docs/venv ]]; then
-  echo "ERROR: venv directory does not exist. First run 'bin/env-var-docs-create-venv' then run this script."
-  exit 1
-fi
-
-source env-var-docs/venv/bin/activate
-
-export APPLICATION_CONF_PATH=server/conf/application.conf
-export ENV_VAR_DOCS_PATH=server/conf/env-var-docs.json
-
-echo "Running env-var-docs/check_vars_documented.py..."
-python env-var-docs/check_vars_documented.py
-
-echo "Running env-var-docs/run_regex_tests.py..."
-python env-var-docs/run_regex_tests.py
+docker run --rm \
+  -v "$(pwd):/code" \
+  -e APPLICATION_CONF_PATH=server/conf/application.conf \
+  -e ENV_VAR_DOCS_PATH=server/conf/env-var-docs.json \
+  python:3.11.4-slim \
+  sh -c 'cd code && \
+  pip install -e env-var-docs/parser-package[test] && \
```

Comment:
**suggestion (blocking):** 

1. create a named docker container for this for clarity and so that different invocations of these scripts reuse the same container, otherwise they'll be slow as they re-run `pip install`.
1. write a docker compose file that specifies the environment variables and code paths needed for the container
1. invoke docker through shell functions specific to this new container/compose file, following the patterns established in https://github.com/civiform/civiform/blob/main/bin/lib/docker.sh